x-pack/plugins/lens/public/datasources/form_based/operations/definitions/index.ts

```diff
@@ -558,7 +558,8 @@ interface FieldBasedOperationDefinition<C extends BaseIndexPatternColumn, P = {}
     layer: FormBasedLayer,
     uiSettings: IUiSettingsClient,
     orderedColumnIds: string[],
-    operationDefinitionMap?: Record<string, GenericOperationDefinition>
+    operationDefinitionMap?: Record<string, GenericOperationDefinition>,
+    colId?: string
```

Comment:
the `columnId` parameter above passed to the field operations toEsAggs() function is not the actual column id on the layer but the id which should be used in aggregations.

Reply:
nit: can it be renamed to `actualColId` and maybe leave this comment in the function description above?